include/libtorrent/torrent_status.hpp

```diff
@@ -545,7 +546,7 @@ namespace libtorrent
 		bool stop_when_ready = false;
 
 		// the info-hash for this torrent
-		sha1_hash info_hash{nullptr};
```

Comment:
? I seem to recall that the default constructor leaves the ``sha1_hash`` uninitialized.